backend/Services/AuthService.cs

```diff
@@ -39,5 +40,10 @@ public bool IsValidLoginInfo(string username, string password)
                 return false;
             }
         }
+
+        public bool IsExistingUsername (string username)
+        {
+            return _users.GetByUsername(username) != null;
+        }
```

Comment:
Move this logic into the `UserService` to be used inside `Create()` (I don't think it even needs to be a separate function)